sirepo/sim_data/__init__.py

```diff
@@ -622,6 +622,26 @@ def schema(cls):
     def sim_file_basenames(cls, data):
         return cls._sim_file_basenames(data)
 
+    # TODO(mvk):  use a server to provide access to sim db and library files
+    @classmethod
+    def _sim_file_to_other_sim_lib(
+        cls, sim_id, basename, other_sim_type, qcall=None, model_name=None, field=None
+    ):
+        cls._assert_server_side()
```

Comment:
I don't see how this would work. It's not server side.

Reply:
It does work, though - `_cfg.lib_file_uri` is `None`. Does this imply something deeper is amiss?

Reply:
Yes indeed https://github.com/radiasoft/sirepo/issues/5254

Reply:
Obviated